use/wordcount-example.md

```diff
@@ -0,0 +1,334 @@
+---
+layout: default
+title: "Beam WordCount Example"
+permalink: /use/wordcount-example/
+redirect_from: /docs/use/wordcount-example/
+---
+
+# Apache Beam WordCount Example
+
+> **Note:** This walkthrough is still in progress. There is an open issue to finish the walkthrough ([BEAM-664](https://issues.apache.org/jira/browse/BEAM-664)).
+
+The WordCount examples demonstrate how to set up a processing pipeline that can read text, tokenize the text lines into individual words, and perform a frequency count on each of those words. The Beam SDKs contain a series of these four successively more detailed WordCount examples that build on each other. The input text for all the examples is a set of Shakespeare's texts.
+
+Each WordCount example introduces different concepts in the Dataflow SDKs. Begin by understanding Minimal WordCount, the simplest of the examples. Once you feel comfortable with the basic principles in building a pipeline, continue on to learn more concepts in the other examples.
+
+* **Minimal WordCount** demonstrates the basic principles involved in building a Dataflow pipeline.
```

Comment:
Ditto on Dataflow. Probably should be 0 Dataflow mentions at this point. (Will add some when you get to the per-runner instructions.)